.ci_support/pyiron_config.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+
+def main():
+    current_path = os.path.abspath(os.path.curdir)
+    top_level_path = current_path.replace('\\', '/')
+    resource_path = os.path.join(current_path, "pyiron", "resources").replace('\\', '/')
+    pyiron_config = os.path.expanduser('~/.pyiron').replace('\\', '/')
+    # conda_prefix = os.getenv('CONDA')
+    # additional_resource = os.path.join(conda_prefix, "share", "pyiron").replace('\\', '/')
+    # if os.path.exists(additional_resource):
+    #     resource_path = resource_path + ", " + additional_resource
+    
+    print(f"resource_path:{resource_path}")
```

Comment:
I guess this is still a development print?

Reply:
I removed it.